src/utils/qrHelpers.js

```diff
@@ -52,6 +52,20 @@ const extractImmunizations = (bundle) => {
   return immunizationResources;
 };
 
+const filterDuplicateImmunizations = (resources) => {
+  // filter immunization resources to those with unique vaccine codes/occurrence dates
+  const filteredResources = resources.filter(
+    (r, index, self) => r.resource.resourceType !== 'Immunization'
+      || self.findIndex(
+        (e) => e.resource.occurrenceDateTime
+        === r.resource.occurrenceDateTime
+        && e.resource.vaccineCode.coding[0].code.coding
```

Comment:
vaccineCode.coding[0].code.coding is undefined - this will filter out any 2 immunizations with the same occurenceDateTime. It should be vaccineCode.coding[0].code. 
Should add a unit test with same date and different code, as well as same code but different date.